junebug/tests/helpers.py

```diff
@@ -161,3 +201,28 @@ def get_dispatched_messages(self, queue):
         amqp_client = self.api.message_sender.client
         return amqp_client.broker.get_messages(
             'vumi', queue)
+
+    def assert_was_logged(self, msg):
+        return self.assertTrue(any(
+            msg in log.getMessage()
+            for log in self.logging_handler.buffer))
+
+    def assert_request(self, req, method=None, body=None, headers=None):
+        if method is not None:
+            self.assertEqual(req['request'].method, 'POST')
+
+        if headers is not None:
+            for name, values in headers.iteritems():
+                self.assertEqual(
+                    req['request'].requestHeaders.getRawHeaders(name),
+                    values)
+
+        if body is not None:
+            self.assertEqual(json.loads(req['body']), body)
+
+    def assert_body_contains(self, req, **fields):
+        body = json.loads(req['body'])
+
+        self.assertEqual(
+            dict((k, v) for k, v in body.iteritems()),
+            body)
```

Comment:
I'm a little confused as to what this is actually doing. Is it not just comparing `body` to itself? Should we be comparing it to `fields`?